Tools/TestWebKitAPI/Tests/WebCore/PushDatabase.cpp

```diff
@@ -284,6 +290,7 @@ class PushDatabaseTest : public testing::Test {
         .sharedAuthSecret = { 24 }
     };
     PushRecord record7 {
+        .identifier = LegacyNullableObjectIdentifier<PushSubscriptionIdentifierType>(7),
```

Comment:
These should probably all be 0 to match the existing behavior. When you insert a record we ignore the identifier, it gets populated when returned as an insert result since the identifier is basically coming from an auto-increment column in the DB.

Reply:
Switched to using a pragma to ignore the "uninitialized" fields.